### DISCUSSION TITLE
Neper -S error: Failed to parse directory

### DISCUSSION THREAD

**Discussion by Yuanguo1994** [neperfepx/neper#119] [Q&A] [answered]
Hello, guys. It's me again. I have problem in using neper S mode to postprocess the simulation result.
I have read through the nper manual, but still feel confused about the S mode. In the book, the command should be like this 'neper -S fepx_res_dir'. 'fepx_res_dir' should be the raw data directory. Does it mean I should type the file path of the raw data for 'fepx_res_dir'? Actually, I have tried, but it failed as follows:

```
yt2n19@UOS-218102:/mnt/c/Ubuntu-app$ neper -S 2_triaxCSR

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.1.2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 1 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/yt2n19/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -S loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] 2_triaxCSR
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Error  :   - Failed to parse directory.
Aborted (core dumped) 
```

Could you please give me some guidance?

**Comment by mkasemer** [comment, marked as answer]
This is a path issue, you must provide the precise location of the directory which contains the output. It seems you are in your top-level directory (`Ubuntu-app`), and to run the command you provided, you must be in the directory directly above `2_triaxCSR`. Alternatively, you could provide the path to the `2_triaxCSR` in your `neper -S` command.

Example 1: You are in `/home/path/to/examples/` which contains the folder `2_triaxCSR`. You can simply run `neper -S 2_triaxCSR`.

Example 2: You are currently in `/home/`. You would run `neper -S /home/path/to/examples/2_triaxCSR/`.
